CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## Unreleased
 
+## 0.17.0
+* Updated dependencies:
```

Comment:
Just realized that I missed this as a CL entry a bit ago: https://github.com/hashicorp/vault-plugin-auth-jwt/pull/240

If you could add that as an "improvement", that would be great. Thanks!

Reply:
Sure thing! Added in [ed90e15](https://github.com/hashicorp/vault-plugin-auth-jwt/pull/249/commits/ed90e154aa208617d2eb7bc2fb1f831887fbebde)